doc/material-controls-extensions.md

```diff
@@ -4,31 +4,61 @@ uid: Uno.Themes.Material.Extensions
 
 # Material Control Extensions
 
+> [!NOTE]
+> Uno Material also has support for C# Markup through a [Uno.Material.WinUI.Markup](https://www.nuget.org/packages/Uno.Material.WinUI.Markup) NuGet Package. To get started with Uno Material in your C# Markup application, add the `Uno.Material.WinUI.Markup` NuGet package to your **App Code Library** project and your platform heads.

```

Comment:
We can link back to the Getting Started section that mention C# Markup with an xref here

Reply:
Huum, I removed the [Using C# Markup](https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html#using-c-markup) section from that page, since all the C# Markup code is now integrated in the tabs with the Xaml ones, and added a similar note to that page instead:

https://github.com/unoplatform/Uno.Themes/blob/931cbd54c9a2bb809399a90b1b6fce677319279d/doc/material-getting-started.md?plain=1#L37-L38

Do you think it's better to bring back that section? 

@kazo0